rpm/0004-Introduce-mount-sd-service-that-is-executed-as-user.patch

```diff
@@ -93,9 +93,9 @@ index 00000000..04a745d2
 +++ b/tools/udisksctl-user
 @@ -0,0 +1,4 @@
 +#!/bin/sh
-+ARGS=$@
-+DEVICEUSER=$(loginctl list-sessions | grep seat0 | tr -s " " | cut -d " " -f 4)
-+/bin/su -l $DEVICEUSER -c "/usr/bin/udisksctl $ARGS"
++ARGS="$@"
```

Comment:
- ARGS is redundant you can just use plain "$@".

Reply:
???
`ARGS` is not at all "redundant" and I do use `"$@"`!
What is your point here?

Reply:
You can use "$@" directly the ARGS variable is redundant.

You can just do `/bin/su -l "$DEVICEUSER" -c "/usr/bin/udisksctl "$@"`.

Reply:
True, but that would be beautifying code written by Jolla, which ought to be done by Jolla / sailors.

I only care about fixing apparent flaws.  YMMV, because you are paid for working on this code.